apps/prairielearn/assets/scripts/question.ts

```diff
@@ -164,3 +170,58 @@ function updateStatus(submission) {
   }
   display.innerHTML = label;
 }
+
+function setupDynamicObjects() {
+  // Install on page load and reinstall on websocket re-render
+  document.querySelectorAll('a.disable-on-click').forEach((link) => {
+    link.addEventListener('click', () => {
+      link.classList.add('disabled');
+    });
+  });
+  // Enable popover
+  $('[data-toggle="popover"]').popover({ sanitize: false, container: 'body' });
+
+  if (document.getElementById('submission-suspended-data')) {
+    const countdownData = decodeData<{
+      serverTimeLimitMS: number;
+      serverRemainingMS: number;
+    }>('submission-suspended-data');
+    setupCountdown({
```

Comment:
Indeed this is replaced. The elements with `submission-suspended-data` are in the question footer, which is added with the socket results. Note that the timer is only shown if the result is successful, since an invalid submission does not count towards the grade rate.